src/boot/efi/boot.c

```diff
@@ -1004,6 +1004,16 @@ static bool menu_run(
                                 status = xstrdup16(u"Reboot into firmware interface not supported.");
                         break;
 
+                case KEYPRESS(0, 0, 'O'):
+                case KEYPRESS(0, 0, 'o'):
```

Comment:
Urgh, I'm gonna fat-finger this so hard when going for `p`. Can't you pick something else? I'd rather have `Ctrl+o` and `Ctrl+b` or so.